### ISSUE TITLE
Release 4.9.0 - Alpha 2 - Service

### ISSUE THREAD

**Issue by wazuhci** [wazuh/wazuh#24580]
### Packages tests metrics information

|||
| :--                              | :--                                                             |
| **Main release stage issue**     | #24573                                         |
| **Main packages metrics issue**  | #24575                                                   |
| **Version**                      | 4.9.0                                                   |
| **Release stage**                | Alpha 2                                                   |
| **Tag**                          | https://github.com/wazuh/wazuh/tree/v4.9.0-alpha2          |

---

| System        | Status | Build |
| :--           | :--:   | :--  |
| CentOS 6      | :red_circle:     | https://ci.wazuh.info/job/Test_service/7960/   |
| CentOS 7      | :red_circle:     | https://ci.wazuh.info/job/Test_service/7961/   |
| Ubuntu Trusty | :green_circle:     | https://ci.wazuh.info/job/Test_service/7962/   |
| Ubuntu Bionic | :green_circle:     | https://ci.wazuh.info/job/Test_service/7963/   |

---

Status legend:

:black_circle: - Not started/In progress
:white_circle: - Skipped
:red_circle: - Found new errors/bugs/failures
:yellow_circle: - Found known errors/bugs/failures
:green_circle: - All tests passed

---

## Conclusion 🟡 

Known errors detected, fixed on v4.8.1 yet not merged on v4.9.0.

- https://github.com/wazuh/wazuh/issues/24503 

---

## Auditor's validation

In order to close and proceed with the release or the next candidate version, the following auditors must give the green light to this RC.

- [ ] @davidjiglesias

---

**Comment by QU3B1M** [comment]
### On hold 🔐 

Release tests are on hold until new order

**Comment by rauldpm** [comment]
This test can be continued

**Comment by pcapellan** [comment]
# Analysis report 

## CentOS 6 :red_circle: 
 
```
08:40:53  Failure talking to yum: Cannot find a valid baseurl for repo: centos-sclo-sclo
```
Issue related:
- https://github.com/wazuh/wazuh/issues/24503

Related:  https://github.com/wazuh/wazuh-jenkins/issues/6700

## CentOS 7 :red_circle: 

```
05:39:52  Failure talking to yum: Cannot find a valid baseurl for repo: centos-sclo-sclo
```

Related: https://github.com/wazuh/wazuh-jenkins/issues/6700

**Comment by QU3B1M** [comment]
LGTM!

**Comment by rauldpm** [comment]
LGTM 

CentOS 6 and CentOS 7 should be fixed when we merge 4.8.1 into 4.9.0
